content/options.html

```diff
@@ -22,7 +22,45 @@
 
   <label for="github-username">Github Username</label>
   <input type="text" id="github-username" data-type="ghuser">
+
+  <h1>Working Hours</h1>
+  <br>
+
+  <label for="working-hours-checkbox">Only show review count during working hours</label>
+  <input type="checkbox" id="working-hours-checkbox">
```

Comment:
Could you please put the checkbox before the label? Perhaps even inside the <label> itself.